tests/components/ipp/test_init.py

```diff
@@ -12,8 +14,12 @@ async def test_config_entry_not_ready(
     hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
 ) -> None:
     """Test the IPP configuration entry not ready."""
-    entry = await init_integration(hass, aioclient_mock, conn_error=True)
-    assert entry.state is ConfigEntryState.SETUP_RETRY
+    with patch(
+        "homeassistant.components.ipp.coordinator.IPPDataUpdateCoordinator.update_more_info_url",
```

Comment:
We can't patch that here. Then no tests are being made of the new code. It needs to mock the actual return.

Reply:
Do the existing bin files have the correct info in them or do I need to make a new one?